iron/release-rhel-build.yaml

```
@@ -25,6 +25,7 @@ package_blacklist:
 - fuse_constraints  # Requires unreleased changes
 - gazebo_dev  # Gazebo has no RPM package
 - geometric_shapes  # assimp has no RPM packages for RHEL 9
+- gz_ros2_control # Gazebo has no RPM package
```

To please the YAML linter, there should be two spaces before a comment.

```suggestion
- gz_ros2_control  # Gazebo has no RPM package
```